crates/core/tedge/src/cli/mqtt/subscribe.rs

```diff
@@ -108,6 +110,10 @@ fn subscribe(cmd: &MqttSubscribeCommand) -> Result<(), MqttError> {
                 eprintln!("INFO: Disconnected");
                 break;
             }
+            Ok(Event::Outgoing(Outgoing::Disconnect)) => {
+                eprintln!("INFO: Interrupted");
```

Comment:
```suggestion
                eprintln!("ERROR: Interrupted");
```

Reply:
I wouldn't class this log message as an ERROR as it is normal behaviour for the user to want to stop the subscriber and the expected way is to use Ctrl-c

Reply:
I even wonder to get rid of this INFO message.

I tried to be consistent with the messages already emitted by this command (`Connected`, `Disconnect`, and now `Interrupted`). But are these really useful?

Reply:
> I even wonder to get rid of this INFO message.

I vote +1 for it.

Reply:
Yeah, let's get rid of the log message. I just checked what `mosquitto_sub` does, and it does not print out a log message about being killed.

Reply:
Removed by https://github.com/thin-edge/thin-edge.io/pull/2936/commits/53d65369d5544b0e320a978b80863cd6a2d9b8dc